CMS-Enterprise/mint-app
# [MINT-2506] Daily Digest Updates with Data Exchange Approach

# MINT-2506

## Description


Added analyzed audit daily digest support for Data Exchange Approach table and Marked Complete logic


## How to test this change

1. Start up services with Faktory enabled
2. Create a Data Exchange Approach
3. Update Data Exchange Approach and mark it complete
4. Verify email received via MailCatcher

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
